modules/basicfuncs/tests/test_basicfuncs.c

```diff
@@ -367,6 +367,52 @@ Test(basicfuncs, test_list_funcs)
   assert_template_format("$(list-append foo,bar,baz 'xxx,')", "foo,bar,baz,\"xxx,\"");
   assert_template_format("$(list-append foo,bar,baz 'a\tb')", "foo,bar,baz,\"a\\tb\"");
 
+  assert_template_format("$(list-search)", "");
+  assert_template_format("$(list-search almafa)", "");
+  assert_template_format("$(list-search almafa 0)", "");
```

Comment:
I would consider those as incorrect call, as searching nothing sure could mean nothing, but most likely an user error this is.

Reply:
Thanks for the heads-up! I added error messages, and removed the invalid calls from here.